src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/index.tsx

```diff
@@ -372,15 +373,17 @@ class Breadcrumbs extends React.Component<Props, State> {
         isCentered
       >
         {filteredBySearch.length > 0 ? (
-          <List
-            breadcrumbs={filteredBySearch}
-            event={event}
-            orgId={orgId}
-            onSwitchTimeFormat={this.handleSwitchTimeFormat}
-            displayRelativeTime={displayRelativeTime}
-            searchTerm={searchTerm}
-            relativeTime={relativeTime!} // relativeTime has to be always available, as the last item timestamp is the event created time
-          />
+          <ErrorBoundary>
```

Comment:
cc @priscilawebdev

Reply:
oh thanks for pointing it out...let me check it